examples/scenarioSepMomentumManagement.py

```diff
@@ -347,6 +347,8 @@ def run(momentumManagement, cmEstimation, showPlots):
     
     # Set up the SRP dynamic effector
     SRP = facetSRPDynamicEffector.FacetSRPDynamicEffector()
+    SRP.numFacets = 10
+    SRP.numArticulatedFacets = 0
```

Comment:
If I have rotating solar arrays, shouldn't it be `SRP.numArticulatedFacets = 4`? 
Also, based on your count of 10 I'm assuming each array is counted twice (front and rear facet), plus the 6 facets of the spacecraft. Can't the solar arrays be modeled as a single facet though, given that they're approximately disks?